webhook_server/libs/handlers/runner_handler.py

```
@@ -742,3 +826,50 @@ async def cherry_pick(self, pull_request: PullRequest, target_branch: str, revie
                 pull_request.create_issue_comment, f"Cherry-picked PR {pull_request.title} into {target_branch}"
             )
+
+    async def run_retests(self, supported_retests: list[str], pull_request: PullRequest) -> None:
+        """Run the specified retests for a pull request.
+
+        Args:
+            supported_retests: List of test names to run (e.g., ['tox', 'pre-commit'])
+            pull_request: The PullRequest object to run tests for
+        """
+        if not supported_retests:
+            self.logger.debug(f"{self.log_prefix} No retests to run")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)